support-services/datafeeder/datadir/default.properties

```diff
@@ -0,0 +1,27 @@
+domainName=georchestra.test.org
```

Comment:
I think you should keep the original contents of defaults.properties, as it will be easier in the future to keep them in sync.